custom/icds_reports/utils/aggregation_helpers/ls_awc_visit_form.py

```diff
@@ -57,12 +59,12 @@ def aggregate_query(self):
             SELECT
             state_id,
             location_id as supervisor_id,
-            %(month)s::DATE AS month,
+            %(start_date)s::DATE AS month,
             count(distinct awc_id) as unique_awc_vists
                 FROM "{ucr_tablename}"
                 WHERE submitted_on > %(start_date)s AND  submitted_on< %(end_date)s
                 AND location_entered is not null and location_entered <> ''
-                GROUP BY location_id
+                GROUP BY state_id,location_id
```

Comment:
is this just a sql syntax thing, or do we really want to group by state

Reply:
yeah its just because of sql syntax constraint state_id is in group_by. though making no harm to result

Reply:
👍